src/olympia/constants/search.py

```diff
@@ -21,7 +21,7 @@
     'danish': ['da'],
     'german': ['de'],
     'greek': ['el'],
-    'english': ['en-us'],
+    'english': ['en-us', 'en-gb'],
```

Comment:
So, wait... what happens when we try indexing an add-on with both an `en-us` and an `en-gb` translation?